### PR TITLE
amp-fx-collection: documentation

### PR DESCRIPTION
**Pull request by aghassemi**

### CODE REVIEW

File: extensions/amp-fx-collection/amp-fx-collection.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <td width="40%"><strong>Description</strong></td>
+    <td>Provides a collection of preset visual effects such as parallax</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-fx-collection" src="https://cdn.ampproject.org/v0/amp-fx-collection-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>None</td>
```
Comment (cathyxz) [inline extensions/amp-fx-collection/amp-fx-collection.md]: None or `nodisplay`? I've seen `nodisplay` a couple of times, e.g. in `amp-image-lightbox` that I'm currently heavily referencing...

File: extensions/amp-fx-collection/amp-fx-collection.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>See AMP By Example's <a href="https://ampbyexample.com/components/amp-fx-collection/">annotated amp-fx-collection</a> example.</td>
```
Comment (bpaduch) [inline extensions/amp-fx-collection/amp-fx-collection.md]: 404.  If there's no example, consider removing the row until there is one.

File: extensions/amp-fx-collection/amp-fx-collection.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+##### data-parallax-factor
+
+Specifies a decimal value that controls how much faster or slower the element scrolls
+relative to the scrolling speed:
+
+- A value greater than 1 scrolls the element upward (element scrolls faster) when the user scrolls down the page.
+- A value less than 1 scrolls the element downward (element scrolls slower) when the user scrolls downward.
+- A value of 1 behaves normally.
+- A value of 0 effectively makes the element scroll fixed with the page.
+
```
Comment (bpaduch) [inline extensions/amp-fx-collection/amp-fx-collection.md]: Please add a validation section (that's standard for component docs) (example: https://github.com/ampproject/amphtml/blob/master/extensions/amp-video/amp-video.md#validation).

Comment (bpaduch) [inline extensions/amp-fx-collection/amp-fx-collection.md]: As this component uses `common-extension-attrs`, please include the section for "common attributes", like in this component -> https://github.com/ampproject/amphtml/blob/master/extensions/amp-video/amp-video.md#common-attributes

Comment (aghassemi) [reply extensions/amp-fx-collection/amp-fx-collection.md]: it does not use any of the common attributes ( actually it is not a components itself, the extensions allows the use of `amp-fx` attribute. There is no `<amp-fx-collection>` tag per say)

Comment (aghassemi) [reply extensions/amp-fx-collection/amp-fx-collection.md]: done

File: extensions/amp-fx-collection/amp-fx-collection.md (hunk 1 above)
Comment (aghassemi) [reply extensions/amp-fx-collection/amp-fx-collection.md]: done

**Comment by aghassemi** [comment]
@bpaduch all done, PTAL

**Pull request merged**